src/Microsoft.ML.TensorFlow/TensorFlowModelInfo.cs

```diff
@@ -43,7 +43,7 @@ internal TensorFlowModelInfo(IHostEnvironment env, TFSession session, string mod
         /// <summary>
         /// Get <see cref="Schema"/> for complete model. Every node in the TensorFlow model will be included in the <see cref="Schema"/> object.
         /// </summary>
-        internal Schema GetModelSchema()
+        public Schema GetModelSchema()
```

Comment:
>public [](start = 8, length = 6)

This need to be public. Its helpful to get the schema without loading model again.